### ISSUE TITLE
Is this project still being maintained?

### ISSUE THREAD

**Issue by gjmveloso** [bmatzelle/gow#224]
Hi There,

Just found this project and I'm wondering if it's still being maintained. The latest release is more than two years old.

Thanks,
Gustavo

**Comment by bmatzelle** [comment]
Yes.  Badly.  I barely use Windows anymore so it's always my last priority.

**Comment by Almenon** [comment]
definitley not being maintained anymore - which is too bad, I like being able to use ls in the windows shell and would love to see gow expanded upon.

Anyone know of any good forks?
